app/javascript/styles/_studyOverview.scss

```diff
@@ -0,0 +1,8 @@
+.dual-plot {
+  float: left;
+  margin-left: 0;
+}
```

Comment:
I don't think this will work in the case of >2 plots, only the last one will get left margin.  I think the structure you want is

```
.dual-plot { 
  float: right;
  margin-left: 60px;
}
.dual-plot:first-of-type {
  float: left;
  margin-left: 0;
}
```

Incidentally, is 'dual-plot' the right class name for this, given that more than two are possible?  Should it be "multi-plot"?

Reply:
Thanks, that makes sense.

For at least this quarter, only <= 2 plots are supported in the default view for the Explore tab.  But your suggestion would ease support for > 2 if/when we pursue that, so I'll incorporate that.